.github/workflows/ci.yml

```diff
@@ -68,7 +68,7 @@ jobs:
       run: npx eslint --max-warnings 0 .
     - name: Build frontend
       working-directory: frontend
-      run: npx webpack --mode=${{ steps.build_mode.outputs.webpack }} --display-error-details
+      run: npx webpack --mode=${{ steps.build_mode.outputs.webpack }}
```

Comment:
The CLI option was removed but I couldn't find anything about it in the changelog. I also couldn't find a good replacement. There is `--stats <value>` but none of the possible values seem to be what we want here. So I decided just to drop the flag. If it turns out to be a problem, we can find a fitting flag then, when we actually have a problem at hand.

Reply:
:shrug: